test/components/Tablo.typecheck.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
```

Comment:
is this file intended to make the build fails if Tablo doesn't typecheck?

Reply:
ups, I was using this to test `Tablo`'s types in development, it wasn't supposed to be committed